src/plugins/LADTable/components/LADRow.vue

```diff
@@ -26,7 +26,14 @@
     @click="clickedRow"
     @contextmenu.prevent="showContextMenu"
   >
-    <td class="js-first-data">{{ domainObject.name }}</td>
+    <td
+      ref="tableCell"
+      class="js-first-data"
+      @mouseover.ctrl="showToolTip"
+      @mouseleave="hideToolTip"
+    >
+      {{ domainObject.name }}
```

Comment:
I had to add spacing here because of linting rules, which resulted in needing to trim the spaces from a test.